test/libsolidity/Behaviour.h

```diff
@@ -0,0 +1,52 @@
+/*
+	This file is part of solidity.
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "Builtin.h"
+#include <memory>
+#include <vector>
+
+namespace solidity::frontend::test
+{
+
+class TestFunctionCall;
+
+class Behaviour
+{
+public:
+	Behaviour() = default;
+	virtual ~Behaviour() = default;
+
+	virtual void begin() {}
```

Comment:
What exactly is begun here? `beginTestCase`? `onTestCaseStart`?

Reply:
True, it makes sense to be more specific here.

Reply:
I choosed `onTestCaseStart` first, but I think `beginTestCase` would be more consistent. Will change that.